packages/mui-system/src/createBox.d.ts

```diff
@@ -1,7 +1,12 @@
-import Box from './Box';
+import { OverridableComponent } from '@mui/types';
+import { BoxTypeMap } from './Box';
+import { Theme as SystemTheme } from './createTheme';
 
-export default function createBox(options?: {
+export default function createBox<
+  Theme extends object = SystemTheme,
+  AdditionalProps extends {} = {},
+>(options?: {
   defaultTheme: object;
   defaultClassName?: string;
   generateClassName?: (componentName: string) => string;
-}): typeof Box;
+}): OverridableComponent<BoxTypeMap<AdditionalProps, 'div', Theme>>;
```

Comment:
I don't understand the fail https://app.circleci.com/pipelines/github/mui/material-ui/87707/workflows/999560f8-4507-40b8-980e-b0d6183349a7/jobs/464126 as well throwing "This is the default value for this type parameter, so it can be omitted." when the params depend on generics 😕

Reply:
Same here. Renaming the generic parameter to something else solves the problem. It seems that tslint does something weird.